installer/linux/kopano.logrotate

```diff
@@ -10,116 +18,169 @@
 /var/log/kopano/dagent.log {
         weekly
         missingok
+	create 640 kopano adm
         rotate 52
+	postrotate
+	    if [ -d /run/systemd/system ] && command systemctl >/dev/null 2>&1 && systemctl is-active --quiet kopano-dagent; then
+                systemctl kill --kill-who all --signal=SIGHUP kopano-dagent
```

Comment:
Is there a reason you chose not to use `systemctl reload kopano-dagent`?

Reply:
Yes, then i tested this reloading did not work, that was on a 8.6.8, we can retest the reload on 8.7. 
Im installing 8.7.1 tomorrow, i'll test the reload and update here.

Reply:
The systemd service files currently declare `/bin/kill -HUP $MAINPID`, which explains why SIGHUP is going to just the main processes. (Else we'd have to use `-HUP -$MAINPID`)

When the main process has a file as a log target, then ECLogger_Pipe will be used and the main process propagates SIGHUP to the pipe writer subprocess which will also reload. Since the pipewriter is shared among all dagent's subprocesses, it effectively reloads the log for all worker subprocesses.

Reply:
How is 8.7.x looking?

Reply:
Looks good as it is now, for debian (and related), reload works fine atm.
I checked what im using now what is here on git. 
I would only suggest, to change: 

create 640 kopano kopano
to 
create 640 kopano adm

The "adm" group is the default debian group with read only rights on the logs. ( where possible)

Reply:
@jengelh , im not sure it the commit change is correctly dont, so if i did something wrong here, please tell me so i know for the next time.  Git is still bit new for me.